python313Packages.pylamarzocco: 2.1.0 -> 2.1.1

fabaff opened the pull request:
Changelog: https://github.com/zweckj/pylamarzocco/releases/tag/v2.1.0


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform:
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- Tested, as applicable:
  - [ ] [NixOS tests] in [nixos/tests].
  - [ ] [Package tests] at `passthru.tests`.
  - [ ] Tests in [lib/tests] or [pkgs/test] for functions and "core" functionality.
- [ ] Ran `nixpkgs-review` on this PR. See [nixpkgs-review usage].
- [ ] Tested basic functionality of all binary files, usually in `./result/bin/`.
- Nixpkgs Release Notes
  - [ ] Package update: when the change is major or breaking.
- NixOS Release Notes
  - [ ] Module addition: when adding a new NixOS module.
  - [ ] Module update: when the change is significant.
- [ ] Fits [CONTRIBUTING.md], [pkgs/README.md], [maintainers/README.md] and other READMEs.

[NixOS tests]: https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests
[Package tests]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests
[nixpkgs-review usage]: https://github.com/Mic92/nixpkgs-review#usage

[CONTRIBUTING.md]: https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md
[lib/tests]: https://github.com/NixOS/nixpkgs/blob/master/lib/tests
[maintainers/README.md]: https://github.com/NixOS/nixpkgs/blob/master/maintainers/README.md
[nixos/tests]: https://github.com/NixOS/nixpkgs/blob/master/nixos/tests
[pkgs/README.md]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md
[pkgs/test]: https://github.com/NixOS/nixpkgs/blob/master/pkgs/test

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 443486`
Commit: `a004b819156f1da78ff534abe1e2c165a4e1c15d`

---
### `x86_64-linux`
<details>
  <summary>:x: 1 package failed to build:</summary>
  <ul>
    <li>home-assistant-component-tests.lamarzocco</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 4 packages built:</summary>
  <ul>
    <li>python312Packages.pylamarzocco</li>
    <li>python312Packages.pylamarzocco.dist</li>
    <li>python313Packages.pylamarzocco</li>
    <li>python313Packages.pylamarzocco.dist</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>home-assistant-component-tests.lamarzocco</summary>
<pre>FAILED tests/components/lamarzocco/test_select.py::test_pre_brew_infusion_select[Linea Mini] - assert None
FAILED tests/components/lamarzocco/test_select.py::test_smart_standby_mode - assert None
FAILED tests/components/lamarzocco/test_select.py::test_select_errors - assert None
FAILED tests/components/lamarzocco/test_sensor.py::test_sensors - assert []
FAILED tests/components/lamarzocco/test_sensor.py::test_steam_ready_entity_for_all_machines[GS3 AV] - assert None
FAILED tests/components/lamarzocco/test_sensor.py::test_steam_ready_entity_for_all_machines[GS3 MP] - assert None
FAILED tests/components/lamarzocco/test_sensor.py::test_steam_ready_entity_for_all_machines[Linea Mini] - assert None
FAILED tests/components/lamarzocco/test_sensor.py::test_steam_ready_entity_for_all_machines[Linea Micra] - assert None
FAILED tests/components/lamarzocco/test_sensor.py::test_sensors_unavailable_if_machine_off - assert None
FAILED tests/components/lamarzocco/test_switch.py::test_switches - assert []
FAILED tests/components/lamarzocco/test_switch.py::test_switches_actions[-set_power-kwargs0] - homeassistant.exceptions.ServiceNotFound: service_not_found
FAILED tests/components/lamarzocco/test_switch.py::test_switches_actions[_steam_boiler-set_steam-kwargs1] - homeassistant.exceptions.ServiceNotFound: service_not_found
FAILED tests/components/lamarzocco/test_switch.py::test_switches_actions[_smart_standby_enabled-set_smart_standby-kwargs2] - homeassistant.exceptions.ServiceNotFound: service_not_found
FAILED tests/components/lamarzocco/test_switch.py::test_auto_on_off_switches - assert None
FAILED tests/components/lamarzocco/test_switch.py::test_switch_exceptions - assert None
FAILED tests/components/lamarzocco/test_update.py::test_update - assert []
FAILED tests/components/lamarzocco/test_update.py::test_update_process - KeyError: &#x27;result&#x27;
FAILED tests/components/lamarzocco/test_update.py::test_update_error - assert None
FAILED tests/components/lamarzocco/test_update.py::test_update_times_out - assert None
======================== 64 failed, 13 passed in 9.29s =========================</pre>
</details>
</details>

mdaniels5757 reviewed (dismissed):
Commit message (and PR title) are wrong: this is `2.0.11 -> 2.1.0`, not a no-op.

fabaff committed:
python313Packages.pylamarzocco: remove disabled

fabaff committed:
python313Packages.pylamarzocco: 2.1.0 -> 2.1.1

Diff: https://github.com/zweckj/pylamarzocco/compare/v2.1.0...v2.1.1

Changelog: https://github.com/zweckj/pylamarzocco/releases/tag/v2.1.1

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 443486`
Commit: `4a56804f90a07b58187be6d399c5b14efd100baf`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 5 packages built:</summary>
  <ul>
    <li>home-assistant-component-tests.lamarzocco</li>
    <li>python312Packages.pylamarzocco</li>
    <li>python312Packages.pylamarzocco.dist</li>
    <li>python313Packages.pylamarzocco</li>
    <li>python313Packages.pylamarzocco.dist</li>
  </ul>
</details>
